python/linear_regression.qmd

```diff
@@ -0,0 +1,43 @@
+---
+title: "Linear Regression"
+output: html_document
+---
+
+To demonstrate the use of linear regression we examine a dataset that illustrates the relationship between Height and Weight in a group of 237 teen-aged boys and girls. The dataset is available [here](https://github.com/PSIAIMS/CAMIS/blob/3dca6398dca4f242eb0f0c316d7499eaba8adb13/data/htwt.csvhttps://github.com/PSIAIMS/CAMIS/blob/3dca6398dca4f242eb0f0c316d7499eaba8adb13/data/htwt.csv) and is imported to the workspace.
+
+### Descriptive Statistics
+
+The first step is to obtain the simple descriptive statistics for the numeric variables of htwt data, and one-way frequencies for categorical variables. This is accomplished by employing summary function. There are 237 participants who are from 13.9 to 25 years old. It is a cross-sectional study, with each participant having one observation. We can use this data set to examine the relationship of participants' height to their age and sex.
+
+```{python}
+import pandas as pd
+import statsmodels.api as sm
+
+# Importing CSV
+htwt = pd.read_csv("../data/htwt.csv")
+```
+
+In order to create a regression model to demonstrate the relationship between age and height for females, we first need to create a flag variable identifying females and an interaction variable between age and female gender flag.
+
+```{python}
+htwt['female'] = (htwt['SEX'] == 'f').astype(int)
+htwt['fem_age'] = htwt['AGE'] * htwt['female']
+```
```

Comment:
Can you please add a line of code so that we can see a preview of the results of updating those values?

Reply:
Thanks :)